crates/cast/src/cmd/call.rs

```diff
@@ -368,6 +375,17 @@ impl CallArgs {
 
         Ok(Some(state_overrides_builder.build()))
     }
+
+    /// Parse state overrides from command line arguments.
+    pub fn get_block_overrides(&self) -> eyre::Result<Option<BlockOverrides>> {
+        // Early return if no override set - <https://github.com/foundry-rs/foundry/issues/10705>
+        if self.block_time.as_ref().is_none() {
+            return Ok(None);
+        }
+
+        let block_overrides = BlockOverrides::default();
```

Comment:
this works, we'll need to check this for any additional overrides field we will introduce